src/components/product-list-component/ProductList.js

```diff
@@ -24,9 +24,9 @@ const ProductList = () => {
             <div className="product-container">
                 <div className="product-inner-container">
                     {
-                        productdata.slice(1,16).map((curProduct) => {
+                        productdata.slice(1, 16).map((curProduct) => {
                             return (
-                                <Product category={curProduct.category} image={curProduct.image} price={curProduct.price} key={curProduct.id} />
+                                <Product id={curProduct.id} category={curProduct.category} image={curProduct.image} price={curProduct.price} key={curProduct.id} />
```

Comment:
Can we use object props as product instead of primitive props.
https://refactoring.guru/smells/primitive-obsession